src/common/Surface.js

```diff
@@ -130,13 +138,43 @@
             .display(this.showTitle())
             .render()
         ;
+
+        var surfaceButtons = element.selectAll(".surface-button").data(this.buttonAnnotations());
+        surfaceButtons.enter().append("tspan")
```

Comment:
in SVG tspan is only valid within a <text> element.  "g" would be a better choice?